Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
```

Comment:
when you bundle install, it also looks at the gem spec, and there are some dependencies in there